packages/next/src/build/webpack/plugins/define-env-plugin.ts

```diff
@@ -273,6 +273,12 @@ export function getDefineEnv({
     'process.env.__NEXT_LINK_NO_TOUCH_START':
       config.experimental.linkNoTouchStart ?? false,
     'process.env.__NEXT_ASSET_PREFIX': config.assetPrefix,
+    ...(config.experimental.nextUrlServerPrefix
+      ? {
+          'process.env.__NEXT_NEXT_EXPERIMENTAL_NEXT_URL_SERVER_PREFIX':
```

Comment:
Where is this env var assigned?

Also it seems to have 3 `NEXT` in it so I think it can be simplified to:

```suggestion
          'process.env.__NEXT_EXPERIMENTAL_URL_SERVER_PREFIX':
```